dlrover/python/master/node/dist_job_manager.py

```diff
@@ -177,6 +179,7 @@ def __init__(
         )
         self._scaler: Scaler = job_scaler
         self._init_training_node_manager()
+        self._training_node_configure = TrainingNodeConfigure()
```

Comment:
The filed 'self._training_node_configure' should be defined in their parent class 'JobManager' for all the sub class need this field.

Reply:
Training node configure is only valid in distributed mode. It is better to put it here.

Reply:
Later a PR will fix these staff.